src/Microsoft.DocAsCode.Build.Engine/XrefDetails.cs

```diff
@@ -114,7 +114,6 @@ public HtmlAgilityPack.HtmlNode ConvertToHtmlNode(string language)
                 string value = InnerHtml;
                 if (string.IsNullOrEmpty(value))
                 {
-                    value = StringHelper.HtmlEncode(Text);
```

Comment:
yes!

---
In reply to: [83991038](https://github.com/dotnet/docfx/pull/799#discussion_r83991038) [](ancestors = 83991038)